File: fibonacci.py
def fibonacci(n):
    if n <= 1:
        return n
    else:
        return fibonacci(n - 1) + fibonacci(n - 2)


def main():
    print(fibonacci(10))
    print(fibonacci(20))
    print(fibonacci(30))
    print(fibonacci(35))
    print(fibonacci(40))

if __name__ == "__main__":
    main()

File: fibonacci.go
package main

import "fmt"

func fibonacci(n int64) int64 {
  if n <= 1 {
    return n
  } else {
    return fibonacci(n-1) + fibonacci(n-2)
  }
}

func main() {
  fmt.Println(fibonacci(10))
  fmt.Println(fibonacci(20))
  fmt.Println(fibonacci(30))
  fmt.Println(fibonacci(35))
  fmt.Println(fibonacci(40))
}